### DISCUSSION TITLE
Run generation using CLI instead of GUI

### DISCUSSION THREAD

**Discussion by SP-l33t** [lllyasviel/Fooocus#2922] [Q&A] [answered]
Hey there!

Thanks a lof for the project, it looks really good!

Is it possible to run the generation with CLI or Requests? Any links or examples will be highly appreciated.

**Comment by mashb1t** [comment, marked as answer]
Using Fooocus with CLI only is not possible OOTB, and I sadly don't get what you mean by requests (REST API?).
For API usage you can check out the default built-in gradio websockets api or https://github.com/mrhan1993/Fooocus-API as fork with additional API layer (no progress updates compared to websockets).
